src/components/MultiSelect/MultiSelectDialog.tsx

```diff
@@ -0,0 +1,231 @@
+          }}
+          onClear={onClear}
+        />
+        <Box
+          role="dialog"
+          __css={styles.menuContainer}
+          {...(!isOpen && { display: "none" })}
+        >
+          <UnorderedList
+            styleType="none"
+            marginInlineStart="0"
+            __css={styles.menu}
+          >
+            {isOpen &&
+              items.map((item: MultiSelectItem) => (
+                <ListItem key={item.id} py={1}>
+                  {item.children ? (
+                    <>
+                      <Checkbox
+                        id={item.id}
+                        labelText={item.name}
+                        showLabel={true}
+                        name={item.name}
+                        // If onMixedStateChange is not passed as a prop, handle
+                        // the parent checkbox as a regular checkbox using onChange
+                        {...(onMixedStateChange !== undefined
+                          ? {
+                              isChecked: isAllChecked(id, item) || false,
+                              isIndeterminate:
+                                isIndeterminate(id, item) || false,
+                              onChange: onMixedStateChange,
+                            }
+                          : {
+                              isChecked: isChecked(id, item.id) || false,
+                              onChange: onChange,
+                            })}
+                      />
+                      <UnorderedList
+                        styleType="none"
+                        marginInlineStart="0"
+                        __css={styles.menuChildren}
+                      >
+                        {item.children.map((childItem) => {
+                          return (
+                            <ListItem key={childItem.id} py={1}>
+                              <Checkbox
+                                id={childItem.id}
+                                labelText={childItem.name}
+                                showLabel={true}
+                                name={childItem.name}
+                                isChecked={isChecked(id, childItem.id) || false}
+                                onChange={onChange}
+                              />
+                            </ListItem>
+                          );
+                        })}
+                      </UnorderedList>
+                    </>
+                  ) : (
+                    <Checkbox
+                      id={item.id}
+                      labelText={item.name}
+                      showLabel={true}
+                      name={item.name}
+                      isChecked={isChecked(id, item.id) || false}
+                      onChange={onChange}
+                    />
+                  )}
+                </ListItem>
+              ))}
+          </UnorderedList>
+          {isOpen && !isMobile && (
+            <Stack direction="row" spacing={4} justify="flex-end">
```

Comment:
Try to use the NYPL-specific design tokens for spacing rather than the number value. In this case, the number `4` spacing is `"1rem"` which is the same as "s". So this can be `spacing="s"`.

The list of values can be found here: https://github.com/NYPL/nypl-design-system/blob/development/src/theme/foundations/spacing.ts#L72

Reply:
Also, can you try using `ButtonGroup` to see if that works instead of the `Stack` component?